Glossario.json

```diff
@@ -394,6 +399,11 @@
     "acronyms": ["SEV", "SCP", "SPP"],
     "synonyms": ["Story Points Completati", "Story Points Pianificati"]
   },
+  "Sprint Planned Value": {
+	"description": "Valore pianificato per un determinato sprint",
```

Comment:
"Valore pianificato che si intende produrre in un determinato periodo di sviluppo."